docs/website/generator-errors.md

```diff
@@ -50,6 +54,10 @@ This usually indicates a bug in the binding generator; please file a new issue o
 <!-- 1xxx: code generation -->
 <!-- 10xx: errors -->
 
+### <a name='BI1000/>'BI1000: Could not compile the generated API bindings.
+
+This indicates a bug in Xamarin.iOS/Xamarin.Mac; please [submit an issue](https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions).
```

Comment:
` and, if possible, attach the files that the C# compiler failed to compile` ?

Reply:
The generator automatically deletes those files (unless --tmpdir is passed to bgen), so asking customers for those files is probably going to cause more problems than it's worth.